src/services_models/services_constructor.jl

```diff
@@ -169,3 +182,50 @@ function construct_service!(
     smooth_ace_pid!(psi_container, services)
     aux_constraints!(psi_container, sys)
 end
+
+"""
+    Constructs a service for StaticReserveGroup.
+"""
+function construct_service!(
+    psi_container::PSIContainer,
+    services::Vector{SR},
+    ::PSY.System,
+    model::ServiceModel{SR, GroupReserve},
+    ::Dict{Symbol, DeviceModel},
+    ::Vector{<:DataType},
+) where {SR <: PSY.StaticReserveGroup}
+    time_steps = model_time_steps(psi_container)
+    names = (PSY.get_name(s) for s in services)
```

Comment:
You have following the same pattern as the existing code.  However, I think this should be
```
names = [PSY.get_name(s) for s in services]
```
This becomes an axis for a JuMP.DenseAxisArray and I think it would be much more performant to give it an array of strings instead of a generator expression (and make the array more readable).  I actually raised this internally yesterday.

Reply:
@jd-lara Actually, performance doesn't really matter.  I looked at the source and JuMP builds a dictionary out of whatever you pass in for the axes.  But they do store the axes.

I think it's better to change our code to use arrays for two reasons:
1. Readability:  show an array of names and not the entire generator expression
2. I don't think we want the DenseAxisArrays to hold references to our PSY components for their entire lifetime.

Reply:
I changed it for `StaticReserveGroup` :)